lib/ansible/executor/module_common.py

```diff
@@ -50,6 +55,69 @@
 
 # ******************************************************************************
 
+ZIPLOADER_DATA = u'''%(shebang)s
+# -*- coding: utf-8 -*-'
+import os
+import sys
+import base64
+import tempfile
+
+ZIPDATA = """%(zipdata)s"""
+
+def debug(command, zipped_mod):
+    # Okay to use __file__ here because we're running from a kept file
+    basedir = os.path.dirname(__file__)
+    if command == 'expand':
```

Comment:
We talked about htis on slack and decided that since expand/explode overwrites the files, we would either have to leave it as two commands or make the single command check for file existence and not overwrite.  Doing file existence checks seems a bit complex for what's supposed to be a short simple wrapper.  So we'll leave this as two commands.